runtime/caml/signals.h

```diff
@@ -44,6 +44,7 @@ CAMLextern int caml_rev_convert_signal_number (int);
 void caml_execute_signal(int signal_number, int in_signal_handler);
 void caml_record_signal(int signal_number);
 void caml_set_something_to_do (void);
+value caml_check_gc_without_async_callbacks(value root);
```

Comment:
If a finaliser (or other async callback) is pending, then in native code this function will be called on each minor allocation even when the heap is not exhausted.  The code of the function will correctly do nothing in this case, but the comment suggested otherwise.